diff --git a/ranger/rc.conf b/ranger/rc.conf
index b72a117..367d4e9 100644
--- a/ranger/rc.conf
+++ b/ranger/rc.conf
@@ -78,7 +78,7 @@ set preview_images true
 #   while slower, this allows remote previews,
 #   for example during an ssh session.
 #   Tmux is unsupported.
-set preview_images_method ueberzug
+set preview_images_method kitty
 
 # Delay in seconds before displaying an image with the w3m method.
 # Increase it in case of experiencing display corruption.
@@ -463,7 +463,7 @@ map yr copy mode=remove
 map yt copy mode=toggle
 
 map C console compress%space
-# map X console extracthere%space
+map X console extracthere%space
 map cp shell pandoc %f -o %f.pdf
 map ytv console shell youtube-dl -ic%space
 map yta console shell youtube-dl -xic%space
