--- a/core/general_config.vim
+++ b/core/general_config.vim
@@ -60,12 +60,11 @@ if has('wildmenu')
 	"		set wildmode=list:longest,full
 	"		set wildoptions=tagfile
 	" endif
-	      	set wildoptions=pum
 	else
 	      	set nowildmenu
 	      	set wildmode=list:longest,full
 	      	set wildoptions=tagfile
 	endif
 	set wildignorecase
 	set wildignore+=.git,.hg,.svn,.stversions,*.pyc,*.spl,*.o,*.out,*~,%*
 	set wildignore+=*.jpg,*.jpeg,*.png,*.gif,*.zip,**/tmp/**,*.DS_Store
