--- a/core/general_config.vim
+++ b/core/general_config.vim
@@ -59,13 +59,14 @@ if has('wildmenu')
 	"		set nowildmenu
 	"		set wildmode=list:longest,full
 	"		set wildoptions=tagfile
+	" endif
 	if has('nvim')
 	      	set wildoptions=pum
 	else
 	      	set nowildmenu
 	      	set wildmode=list:longest,full
 	      	set wildoptions=tagfile
 	endif
 	set wildignorecase
 	set wildignore+=.git,.hg,.svn,.stversions,*.pyc,*.spl,*.o,*.out,*~,%*
 	set wildignore+=*.jpg,*.jpeg,*.png,*.gif,*.zip,**/tmp/**,*.DS_Store
