--- a/core/general_config.vim
+++ b/core/general_config.vim
@@ -54,13 +54,14 @@ if has('wildmenu')
 	endif
 
 	" if has('nvim')
+	"		set wildoptions=pum
 	if has('nvim')
 	      	set wildoptions=pum
 	else
 	      	set nowildmenu
 	      	set wildmode=list:longest,full
 	      	set wildoptions=tagfile
 	endif
 	set wildignorecase
 	set wildignore+=.git,.hg,.svn,.stversions,*.pyc,*.spl,*.o,*.out,*~,%*
 	set wildignore+=*.jpg,*.jpeg,*.png,*.gif,*.zip,**/tmp/**,*.DS_Store
