diff --git a/themes/agnoster.zsh-theme b/themes/agnoster.zsh-theme
index 5f4efe8..4614b6b 100644
--- a/themes/agnoster.zsh-theme
+++ b/themes/agnoster.zsh-theme
@@ -204,13 +204,15 @@ prompt_hg() {
 
 # Dir: current working directory
 prompt_dir() {
-  prompt_segment blue $CURRENT_FG '%~'
+  #prompt_segment blue $CURRENT_FG '%~'
+  prompt_segment blue $CURRENT_FG '%1d'
 }
 
 # Virtualenv: current working virtualenv
 prompt_virtualenv() {
   if [[ -n "$VIRTUAL_ENV" && -n "$VIRTUAL_ENV_DISABLE_PROMPT" ]]; then
-    prompt_segment blue black "(${VIRTUAL_ENV:t:gs/%/%%})"
+    #prompt_segment blue black "(${VIRTUAL_ENV:t:gs/%/%%})"
+    prompt_segment lightblue black "(${VIRTUAL_ENV:t:gs/%/%%})"
   fi
 }
 
