diff --git a/lua/config/basic/nvim-treesitter.lua b/lua/config/basic/nvim-treesitter.lua
index 1ba6d93..481239c 100644
--- a/lua/config/basic/nvim-treesitter.lua
+++ b/lua/config/basic/nvim-treesitter.lua
@@ -8,7 +8,7 @@ local M = {
         "nvim-treesitter.parsers",
     },
 }
-
+require("nvim-treesitter.install").prefer_git = true
 function M.download_source_settings()
     local default_source = "https://github.com/"
 
diff --git a/lua/config/lsp/nvim-navic.lua b/lua/config/lsp/nvim-navic.lua
index 2a72547..9b3ad01 100644
--- a/lua/config/lsp/nvim-navic.lua
+++ b/lua/config/lsp/nvim-navic.lua
@@ -36,10 +36,12 @@ function M.after()
         {
             callback = function()
                 if not vim.bo.buflisted or vim.tbl_contains(M.ignore_filetype, vim.bo.filetype) then
-                    vim.opt_local.winbar = ""
+                    --vim.opt_local.winbar = ""
+                    vim.opt_local.statusline = ""
                     return
                 end
-                vim.opt_local.winbar = "%{%v:lua.require('nvim-navic').get_location()%}"
+                --vim.opt_local.winbar = "%{%v:lua.require'nvim-navic'.get_location()%}"
+                vim.opt_local.statusline = "%{%v:lua.require'nvim-navic'.get_location()%}"
             end,
         }
     )
diff --git a/lua/core/mapping.lua b/lua/core/mapping.lua
index de60386..8c0d342 100644
--- a/lua/core/mapping.lua
+++ b/lua/core/mapping.lua
@@ -3,6 +3,20 @@ local api = require("utils.api")
 local mapping = {}
 
 api.map.bulk_register({
+    {
+        mode = { "n" },
+        lhs = "Q",
+        rhs = ":q!<cr>",
+        options = { silent = true },
+        description = "quit Neovim",
+    },
+    {
+        mode = { "n" },
+        lhs = "S",
+        rhs = ":w!<cr>",
+        options = { silent = true },
+        description = "save Neovim",
+    },
     {
         mode = { "n" },
         lhs = "<leader><esc>",
diff --git a/spell/en.utf-8.add b/spell/en.utf-8.add
index e198bf0..e69de29 100644
--- a/spell/en.utf-8.add
+++ b/spell/en.utf-8.add
@@ -1,18 +0,0 @@
-#ODO
-TODO/!
-TODO
-agv
-agv
-virtualenv/!
-virtualenv
-CONFIG
-#vim
-#vim/!
-#vim
-nvim/!
-nvim
-modbus
-gzrobot
-config
-api
-DEP
diff --git a/spell/en.utf-8.add.spl b/spell/en.utf-8.add.spl
index 7ee0cd6..e69de29 100644
Binary files a/spell/en.utf-8.add.spl and b/spell/en.utf-8.add.spl differ
