diff --git a/init.vim b/init.vim
index 3a22e6c..dc03e29 100644
--- a/init.vim
+++ b/init.vim
@@ -43,12 +43,12 @@ set number
 set relativenumber
 set cursorline
 set noexpandtab
-set tabstop=2
-set shiftwidth=2
-set softtabstop=2
+set tabstop=4
+set shiftwidth=4
+set softtabstop=4
 set autoindent
-set list
-set listchars=tab:\|\ ,trail:▫
+set nolist
+"set listchars=tab:\|\ ,trail:▫
 set scrolloff=4
 set ttimeoutlen=0
 set notimeout
@@ -106,10 +106,10 @@ augroup NVIMRC
     autocmd BufWritePost *.nvimrc exec ":so %"
 augroup END
 " Undo operations
-noremap l u
+"noremap l u
 " Insert Key
-noremap k i
-noremap K I
+"noremap k i
+"noremap K I
 " Copy to system clipboard
 vnoremap Y "+y
 " Find pair
@@ -135,20 +135,20 @@ inoremap <c-y> <ESC>A {}<ESC>i<CR><ESC>ko
 " < n   i >
 "     e
 "     v
-noremap <silent> u k
-noremap <silent> n h
-noremap <silent> e j
-noremap <silent> i l
-noremap <silent> gu gk
-noremap <silent> ge gj
+"noremap <silent> u k
+"noremap <silent> n h
+"noremap <silent> e j
+"noremap <silent> i l
+"noremap <silent> gu gk
+"noremap <silent> ge gj
 noremap <silent> \v v$h
 " U/E keys for 5 times u/e (faster navigation)
-noremap <silent> U 5k
-noremap <silent> E 5j
+noremap <silent> K 5k
+noremap <silent> J 5j
 " N key: go to the start of the line
-noremap <silent> N 0
+noremap <silent> A 0
 " I key: go to the end of the line
-noremap <silent> I $
+noremap <silent> E $
 " Faster in-line navigation
 noremap W 5w
 noremap B 5b
@@ -181,23 +181,23 @@ cnoremap <M-w> <S-Right>
 " ==================== Window management ====================
 " Use <space> + new arrow keys for moving the cursor around windows
 noremap <LEADER>w <C-w>w
-noremap <LEADER>u <C-w>k
-noremap <LEADER>e <C-w>j
-noremap <LEADER>n <C-w>h
-noremap <LEADER>i <C-w>l
+noremap <LEADER>e <C-w>k
+noremap <LEADER>u <C-w>j
+noremap <LEADER>i <C-w>h
+noremap <LEADER>n <C-w>l
 noremap qf <C-w>o
 " Disable the default s key
 noremap s <nop>
 " split the screens to up (horizontal), down (horizontal), left (vertical), right (vertical)
 noremap su :set nosplitbelow<CR>:split<CR>:set splitbelow<CR>
-noremap se :set splitbelow<CR>:split<CR>
-noremap sn :set nosplitright<CR>:vsplit<CR>:set splitright<CR>
-noremap si :set splitright<CR>:vsplit<CR>
+noremap sd :set splitbelow<CR>:split<CR>
+noremap sl :set nosplitright<CR>:vsplit<CR>:set splitright<CR>
+noremap sr :set splitright<CR>:vsplit<CR>
 " Resize splits with arrow keys
-noremap <up> :res +5<CR>
-noremap <down> :res -5<CR>
-noremap <left> :vertical resize-5<CR>
-noremap <right> :vertical resize+5<CR>
+noremap <M-up> :res +5<CR>
+noremap <M-down> :res -5<CR>
+noremap <M-left> :vertical resize-5<CR>
+noremap <M-right> :vertical resize+5<CR>
 " Place the two screens up and down
 noremap sh <C-w>t<C-w>K
 " Place the two screens side by side
@@ -544,7 +544,7 @@ let g:coc_global_extensions = [
 	\ 'coc-sourcekit',
 	\ 'coc-stylelint',
 	\ 'coc-syntax',
-	\ 'https://github.com/theniceboy/coc-tailwindcss',
+	\ 'coc-tailwindcss',
 	\ 'coc-tasks',
 	\ 'coc-translator',
 	\ 'coc-tsserver',
@@ -1200,4 +1200,5 @@ exec "nohlsearch"
 if has_machine_specific_file == 0
 	exec "e ~/.config/nvim/_machine_specific.vim"
 endif
-
+"au BufWinLeave * silent mkview
+"au BufWinEnter * silent loadview
